homeassistant/components/jewish_calendar/quality_scale.yaml

```diff
@@ -0,0 +1,92 @@
+rules:
+  # Bronze
+  action-setup: done
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
+  docs-actions: done
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: done
+  entity-event-setup:
+    status: exempt
+    comment: Entities of this integration do not explicitly subscribe to events.
+  entity-unique-id: done
+  has-entity-name: done
+  runtime-data: done
+  test-before-configure:
+    status: exempt
+    comment: Local calculation does not require configuration.
+  test-before-setup:
+    status: exempt
+    comment: Local calculation does not require setup.
+  unique-config-entry: done
+
+  # Silver
+  action-exceptions: done
+  config-entry-unloading: done
+  docs-configuration-parameters: done
+  docs-installation-parameters: done
+  entity-unavailable:
+    status: exempt
+    comment: This integration cannot be unavailable since it's a local calculation.
+  integration-owner: done
+  log-when-unavailable:
+    status: exempt
+    comment: This integration cannot be unavailable since it's a local calculation.
+  parallel-updates: done
+  reauthentication-flow:
+    status: exempt
+    comment: This integration does not require reauthentication, since it is a local calculation.
+  test-coverage: done
```

Comment:
```py
async def test_no_discovery_info(
    hass: HomeAssistant, caplog: pytest.LogCaptureFixture
) -> None:
    """Test setup without discovery info."""
    assert BINARY_SENSOR_DOMAIN not in hass.config.components
    assert await async_setup_component(
        hass,
        BINARY_SENSOR_DOMAIN,
        {BINARY_SENSOR_DOMAIN: {CONF_PLATFORM: DOMAIN}},
    )
    await hass.async_block_till_done()
    assert BINARY_SENSOR_DOMAIN in hass.config.components
```
not sure what this tests

Reply:
same for sensor

Reply:
No idea, these were part of one of the original templates. Never bothered understanding it. Maybe from the YAML days?

Reply:
Done